examples/verifying_mfd.py

```diff
@@ -1,16 +1,176 @@
+          "same left-hand side attribute values.\n")
+    print("Let's start by investigating theatres_mfd.csv and "
+          "trying to verify a metric functional dependency in it.\n")
+
+    check_mfd(LHS_INDICES, RHS_INDICES, METRIC, PARAMETERS, TABLE,
+              "[Movie] -> [Duration]")
+
+    print("We learnt that in this case, the specified MFD does not hold, and "
+          "there are two clusters of rows that contain values that prevent "
+          f"our MFD from holding. A {BLUE_CODE}cluster{DEFAULT_COLOR_CODE} "
+          "(with respect to a fixed FD) is a collection of rows that share "
+          "the same left-hand side, but differ in the right-hand side.")
+    print("Let's take a closer look at the indicated clusters.\n")
+    print('"Don Quixote" has been found with 3 different durations: 135, 139 '
+          "and 140. From this, we can conclude that "
+          "(assuming euclidian distance), the maximum distance between "
+          "values is 140 - 135 = 5, which exceeds the provided "
+          "parameter of δ = 3. When we tried to verify the same MFD on this "
+          "dataset with δ = 5, this cluster adhered to the MFD as 5 <= δ, "
+          "and the whole MFD held.")
+    print('"Romeo and Juliet" had the exact same issue: the biggest '
+          "difference between durations of different versions appeared to "
+          "have been 165 - 160 = 5, which meant that the MFD held with "
+          "the greater parameter value of δ = 5.")
+    print("-" * 80)
+
+def mfd_strings():
+    METRIC = "cosine"
+    METRIC_ALGORITHM = "brute"
+    LHS_INDICES = [1]
+    RHS_INDICES = [2]
+    PARAMETERS = [0.75, 0.5, 0.25, 0.1, 0.01]
+    TABLE="examples/datasets/addresses_names.csv"
+    bigrams_table="examples/datasets/bigrams.csv"
+
+    print("Let's showcase this by checking addresses_names.csv and "
+          "trying to verify a metric functional dependency in it.\n")
+
+    check_mfd(LHS_INDICES, RHS_INDICES, METRIC, PARAMETERS, TABLE,
+              "[SSN] -> [Address]", METRIC_ALGORITHM)
+
+    print("To get into the intricacies of how the distance has been "
+          "calculated here, we need to first define what is known as an "
+          f"n-gram. {BLUE_CODE}N-grams{DEFAULT_COLOR_CODE} are collections "
+          "of adjacent symbols of fixed "
+          'length, such as "aab" or "ba". Depending on the length, they are '
+          "called bigrams (2), trigrams (3) and so on.\n")
+    print("For the purposes of illustration, we picked the RHS values from "
+          "the cluster violating the MFD with δ = 0.01 but not δ = 0.1. "
+          "The following table displays the number of occurrences of every "
+          "bigram (columns) occurring in any of these values (rows). "
+          "In total, there are 30 bigrams across two strings.")
+
+    bigrams = pd.read_csv(bigrams_table, header=[0])
+    print(bigrams)
+
+    print("We can now compare those two vectors. For example, we can "
```

Comment:
There was no mention of vectors before.
```suggestion
    print("Let's interpret the rows in the table as coordinates of "
          "vectors x and y respectively. We have ways to compare "
          "the vectors. For example, we can "
```